platform/fabric/core/generic/driver/provider.go

```diff
@@ -31,86 +29,38 @@ type Provider struct {
 	metricsProvider    metrics.Provider
 	channelProvider    generic.ChannelProvider
 	sigService         *sig.Service
-	mspManagerProvider MSPManagerProvider
+	mspManagerProvider identity.MSPManagerProvider
 }
 
 func NewProvider(
```

Comment:
We could have another additional constructor that receives the sigService as a dependency (like we did before), so that other SDKs that want to instantiate a new provider can re-use this one by injecting their own sigservice.